sky/cli.py

```diff
@@ -3557,6 +3557,18 @@ def jobs_logs(name: Optional[str], job_id: Optional[int], follow: bool,
     """Tail the log of a managed job."""
     try:
         if controller:
+            # Get the job_id for the given job name, if specified.
+            if name is not None:
+                job_queue = managed_jobs.queue(refresh=False)
+                job_id = None
+                for job in job_queue:
+                    if job['job_name'] == name:
+                        job_id = job['job_id']
+                        break
+                if job_id is None:
+                    click.echo(f'Managed job with name {name!r} not found.')
+                    sys.exit(1)
```

Comment:
To discuss: will this add too much overhead to the `sky jobs logs`? Now it requires two SSH to the controller IIUC. Also, this seems not used in the smoke test?

Reply:
If a user does `sky jobs logs --controller -n train-job`, then it will also hit this code path?